docs/data/rpc/admin-guide/prerequisites.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Prerequisites
+sidebar_position: 10
+---
+
+The RPC service can be installed on bare metal or a virtual machine. It is natively supported on both Linux and Windows operating systems.
+
+### Hardware Requirements
+
+For deployments expecting up to 1000 requests per second to Stellar RPC, we recommend at least 4GB of RAM and at least a dual core CPU with a 2.4GHz clock speed.
+
+For deployments expecting 1000-10000+ requests per second to Stellar RPC, we recommend at least 8GB of RAM and at least a quad core CPU with a 2.4GHz clock speed.
+
+For all deployments, we recommend at least 10GB of disk/storage space.
```

Comment:
Can we put this content in the actual chart below? Maybe add a "desired RPS" column or something like that, and the RAM value is then different for the 2 rows. 

Unrelated reminder to ourselves to redo these estimates once we're using core's getledgerentries and also when we have classic events being saved/emitted.

Reply:
Related: we should suggest `N` GB of storage per retention day so people can stop asking us to benchmark things.